src/components/dropdown/EllipsisDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Dropdown } from 'react-bootstrap';
+import { DropdownToggleProps } from 'react-bootstrap/esm/DropdownToggle';
+
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+type Props = {
+  header: string;
+  items: { label: string; value: string }[];
+  onClick: (code: string) => void;
+};
+
+type CustomToggleProps = DropdownToggleProps & {
+  onClick: () => void;
+};
+
+const CustomToggle = React.forwardRef<HTMLButtonElement, CustomToggleProps>(
+  ({ onClick }: CustomToggleProps, ref) => (
+    <button
+      className="border-0 bg-transparent"
+      ref={ref}
+      onClick={(e) => {
+        e.preventDefault();
+        onClick(e);
+      }}
+    >
+      <FontAwesomeIcon icon={faEllipsisV} />
+    </button>
+  )
+);
+CustomToggle.displayName = 'CustomToggle';
+
+const EllipsisDropdown = ({ header, items, onClick }: Props) => (
```

Comment:
generalnie nie twórzmy nigdy dwóch komponentów w jednym pliku. Jeśli potrzebujemy stworzyć nowy to zróbmy to w nowym pliku. Jeśli jest to komponent potencjalnie reużywalny w obrębie całej apki to umieśćmy go w /components, a jeśli nie to w bardziej sprecyzowanym miejscu

Reply:
Czuje że to grzech ciężki był, będę pamiętał na przyszłość